src/Behat/Behat/Definition/Printer/ConsoleDefinitionInformationPrinter.php

```diff
@@ -39,12 +39,37 @@ public function setSearchCriterion($criterion)
      * {@inheritdoc}
      */
     public function printDefinitions(Suite $suite, $definitions)
+    {
+        $this->printDefinitionsWithOptionalSuite($definitions, $suite);
+    }
+
+    /**
+     * @param Definition[] $definitions
+     */
+    public function printUnusedDefinitions(array $definitions): void
+    {
+        $unusedDefinitionsText = $this->translateInfoText(
+            'unused_definitions',
+            ['%count%' => count($definitions)]
+        );
+        $this->write('--- ' . $unusedDefinitionsText, true);
+        if (count($definitions) !== 0) {
+            $this->printDefinitionsWithOptionalSuite($definitions);
+        }
+    }
+
+    /**
+     * @param Definition[] $definitions
+     */
+    private function printDefinitionsWithOptionalSuite(array $definitions, ?Suite $suite = null): void
     {
         $search = $this->searchCriterion;
         $output = array();
 
         foreach ($definitions as $definition) {
-            $definition = $this->translateDefinition($suite, $definition);
+            if ($suite) {
```

Comment:
If we don't have a suite, we can't translate the definitions. This is not a problem for the unused definitions functionality, this will be used to find them in the code, so it is best to print them in the original language used in the code